action.yml

```diff
@@ -76,11 +53,43 @@ runs:
         echo "PLAYGROUND_PASSWORD=${{ inputs.PLAYGROUND_PASSWORD }}" >> $GITHUB_ENV
         echo "PLAYGROUND_ID=${{ inputs.PLAYGROUND_ID }}" >> $GITHUB_ENV
         echo "DOCASSEMBLE_DEVELOPER_API_KEY=${{ inputs.DOCASSEMBLE_DEVELOPER_API_KEY }}" >> $GITHUB_ENV
+        echo "ALKILN_VERSION=${{ inputs.ALKILN_VERSION }}" >> $GITHUB_ENV
       shell: bash
     - name: "ALKiln: confirm info"
       run: echo -e "\nRepo is $REPO_URL\nBranch ref is $BRANCH_PATH\nManually added languages are $EXTRA_LANGUAGES\nAll other data is secret\n"
       shell: bash
 
+    # This doesn't use an already established package-lock.json and that
+    # comes with some types of test breaking risks. Same problem as before.
+    - name: Temporarily overwrite or create package.json
+      run: |
+        cat << EOF > "package.json"
+        {
+          "scripts": {
+            "test": "npm run langs_setup && npm run cucumber -- $@",
+            "cucumber_base": "./node_modules/.bin/cucumber-js --require ./node_modules/@suffolklitlab/alkiln/lib/index.js",
+            "cucumber": "run_cucumber(){ npm run cucumber_base -- \"$@\" docassemble/*/data/sources/*.feature; }; run_cucumber",
+            "langs": "npm run langs_setup && ./node_modules/.bin/cucumber-js --require ./node_modules/.bin/alkiln-cucumber docassemble/*/data/sources/*_lang*.feature",
+            "langs_setup": "node_modules/.bin/alkiln-langs 'docassemble/*/data/sources/*.feature'",
+            "setup": "node_modules/.bin/alkiln-setup",
+            "takedown": "node_modules/.bin/alkiln-takedown"
+          },
+          "dependencies": {
+            "colors": "1.4.0",
+            "@cucumber/cucumber": "^7.3.2",
+            "@suffolklitlab/alkiln": "$ALKILN_VERSION"
+          }
+        }
+      shell: bash
+
+    # Install node
+    - name: "ALKiln: Install node packages"
+      uses: actions/setup-node@v1
```

Comment:
The latest [setup-node](https://github.com/actions/setup-node#usage) is version 2

It makes sense that Dependabot does this by default, but if it doesn't it can be configured to do so as well (e.g., https://github.com/codeforamerica/brigade-project-index-statusboard/blob/a288e657777479ea82ce0fa09a056ef956a83ce3/.github/dependabot.yml#L3-L6)

Reply:
#529